src/shared/components/create-case-filters/create-case-filters.component.ts

```diff
@@ -9,7 +9,12 @@ import { DefinitionsService, OrderService } from '../../services';
 
 @Component({
   selector: 'ccd-create-case-filters',
-  templateUrl: './create-case-filters.component.html'
+  templateUrl: './create-case-filters.component.html',
+  styles: [`a.disabled {
```

Comment:
We don't have anywhere else in the codebase that we use this pattern for styles - the mechanism everywhere else looks like this:
`styleUrls: ['./create-case-filters.component.scss']`